modules/dicom_archive/README.md

```diff
@@ -0,0 +1,42 @@
+# Dicom Archive
+
+## Purpose
+
+The DICOM Archive module provides a front end to view the metadata
+about DICOMs which have been inserted into the LORIS pipeline.
```

Comment:
Maybe rephrase "into the LORIS pipeline" as "into the LORIS database via the imaging pipeline"

Reply:
good catch, done.